tests/test_polytracker.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+import os
+from polyprocess import PolyProcess
+
+cwd: str = os.getcwd()
+test_dir: str = cwd + "/tests/"
+bin_dir: str = test_dir + "/bin/"
+test_results_dir: str = bin_dir + "/test_results/"
+bitcode_dir: str = test_dir + "/bitcode/"
+
+"""
+Pytest fixture to init testing env (building tests) 
+
+This runs before any test is executed
+"""
+
+
+@pytest.fixture(scope="session", autouse=True)
+def setup_targets():
+    # Check if bin dir exists
+    if os.path.exists(bin_dir):
+        os.system("rm -r " + bin_dir)
```

Comment:
You will need to add `import subprocess` above.
```suggestion
        subprocess.call(["rm", "-r", bin_dir])
```